server/core/runtime/plan_step_runner_test.go

```diff
@@ -24,358 +25,12 @@ import (
 	. "github.com/runatlantis/atlantis/testing"
 )
 
-func TestRun_NoWorkspaceIn08(t *testing.T) {
```

Comment:
these tests are test for workspace, so moved into `workspace_step_runner_delegate_test.go`